src/common/helper.js

```diff
@@ -440,6 +441,130 @@ async function createResource (challengeId, memberHandle, roleId) {
   return res || false
 }
 
+/**
+ * Create Project
+ * @param {String} name The name
+ * @param {String} description The description
+ * @param {String} type The type
+ * @param {String} token The token
+ * @returns 
+ */
+async function createSelfServiceProject (name, description, type, token) {
+  const projectObj = {
+    name,
+    description,
+    type
+  }
+  const url = `${config.PROJECTS_API_URL}`
+  const res = await axios.post(url, projectObj, {headers: {Authorization: `Bearer ${token}`}})
+  return _.get(res, 'data.id')
+}
+
+/**
+ * Get project payment
+ * @param {String} projectId the project id
+ */
+ async function getProjectPayment (projectId) {
+  const token = await getM2MToken()
+  const url = `${config.CUSTOMER_PAYMENTS_URL}`
+  const res = axios.get(url, {
+    headers: { Authorization: `Bearer ${token}` },
+    params: {
+      referenceId: projectId
+    }
+ })
+  const [payment] = res.data
+  return payment
+}
+
+/**
+ * Charge payment
+ * @param {String} paymentId the payment ID
+ */
+async function capturePayment (paymentId) {
+  const token = await getM2MToken()
```

Comment:
@ThomasKranitsas just in case, to make operations with Customer Payments M2M token need the scope `all:customer-payments`.